recipes/mpc.mdwn

```diff
@@ -52,10 +53,15 @@ updates when the current MPD state changes.
         end
         mpd_widget.text = text
     end
+    local connection
     local function error_handler(err)
         mpd_widget:set_text("Error: " .. tostring(err))
+        -- Try a reconnect soon-ish
+        timer.start_new(60, function()
```

Comment:
I just thought that a disconnect should not happen that often, and if so, it should not wait for a whole minute to reconnect.
Feel free to keep it.